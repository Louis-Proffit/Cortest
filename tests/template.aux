\relax 
\providecommand\babel@aux[2]{}
\@nameuse{bbl@beforestart}
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\abx@aux@refcontext{nty/global//global/global}
\babel@aux{english}{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Modes Longitudinaux}{1}{section.1}\protected@file@percent }
\newlabel{sec:modes-longitudinaux}{{1}{1}{Modes Longitudinaux}{section.1}{}}
\newlabel{sec:modes-longitudinaux@cref}{{[section][1][]1}{[1][1][]1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Profil gaussien}{1}{section.2}\protected@file@percent }
\newlabel{sec:profil-gaussien}{{2}{1}{Profil gaussien}{section.2}{}}
\newlabel{sec:profil-gaussien@cref}{{[section][2][]2}{[1][1][]1}}
\abx@aux@read@bbl@mdfivesum{nohash}
\abx@aux@read@bblrerun
\gdef \@abspage@last{1}
