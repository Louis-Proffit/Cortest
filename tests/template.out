\BOOKMARK [1][-]{section.1}{\376\377\000M\000o\000d\000e\000s\000\040\000L\000o\000n\000g\000i\000t\000u\000d\000i\000n\000a\000u\000x}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000P\000r\000o\000f\000i\000l\000\040\000g\000a\000u\000s\000s\000i\000e\000n}{}% 2
